bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -775,6 +775,57 @@ Allowed values are:
 This option is older than the luhdata options and may be depricated at some point in the future.
 </entry>
 
+<entry id="fates_radiation_model" type="char*256" category="physics"
+        group="clm_inparm" valid_values="norman, twostream" value="norman">
+Set the FATES radiation model
+</entry>
+
+<entry id="fates_hydro_solver" type="char*256" category="physics"
+        group="clm_inparm" valid_values="1D_Taylor, 2D_Picard, 2D_Newton" value="1D_Taylor">
+Set the FATES hydro solver method
+</entry>
+
+<entry id="fates_regeneration_model" type="char*256" category="physics"
+        group="clm_inparm" valid_values="default, trs, trs_no_seed_dyn" value="default">
+Set the FATES seed regeneration model
+Valid values:
+  default: default scheme
+  trs: Tree Recruitment Scheme (Hanbury-Brown et al., 2022)
+  trs_no_seed_dyn: Tree Recruitment Scheme (Hanbury-Brown et al., 2022) without seed dynamics
+</entry>
+
+<entry id="fates_cstarvation_model" type="char*256" category="physics"
+        group="clm_inparm" valid_values="linear, exponential" value="linear">
+Set the FATES carbon starvation model
+</entry>
+
+<entry id="fates_leafresp_model" type="char*256" category="physics"
+        group="clm_inparm" valid_values="ryan1991, atkin2017" value="ryan1991">
+Set the FATES leaf maintenance respiration model
+</entry>
+
+<entry id="fates_stomatal_assimilation" type="char*256" category="physics"
+        group="clm_inparm" valid_values="net, gross" value="net">
+Set net or gross asslimiation for the FATES stomatal model
+</entry>
+
+<entry id="fates_stomatal_model" type="char*256" category="physics"
+        group="clm_inparm" valid_values="ballberry, medlyn" value="ballberry">
+Set the FATES stomatal conductance model
+</entry>
+
```

Comment:
I'm guessing there are no other temperature acclimation models that people are planning on putting in, but I really don't know. It might be safer to leave this as an integer instead of a logical.

Reply:
Thanks for the feedback here.  I had waffled on whether or not to use the integer/name or a logical.  I'll turn it into a named switch.  I'll do something like: `nonacclimating` and `kumarathunge`.

Reply:
I would suggest that for switch values that are author's names, that we try to use the format `nameyear` of the paper describing the parameterization, so `kumarathunge2019`, since it is really the paper that we want to point to, not the author?

Reply:
Yes, I agree we've tried to do that elsewhere also (for example: Ball-Berry1987,Medlyn2011 for stomatalcond_method in BGC). It is the specific paper reference that we do mean in all of this.

Reply:
addressed via [660ed58](https://github.com/ESCOMP/CTSM/pull/2904/commits/660ed58b5b51ee961c2820a884f72d52fff1db77)